mypyc/primitives/list_ops.py

```diff
@@ -27,25 +27,11 @@
     error_kind=ERR_MAGIC,
 )
 
-
-def emit_new(emitter: EmitterInterface, args: List[str], dest: str) -> None:
-    # TODO: This would be better split into multiple smaller ops.
-    emitter.emit_line('%s = PyList_New(%d); ' % (dest, len(args)))
-    emitter.emit_line('if (likely(%s != NULL)) {' % dest)
-    for i, arg in enumerate(args):
-        emitter.emit_line('PyList_SET_ITEM(%s, %s, %s);' % (dest, i, arg))
-    emitter.emit_line('}')
-
-
-# Construct a list from values: [item1, item2, ....]
-new_list_op = custom_op(arg_types=[object_rprimitive],
-                        result_type=list_rprimitive,
-                        is_var_arg=True,
-                        error_kind=ERR_MAGIC,
-                        steals=True,
-                        format_str='{dest} = [{comma_args}]',
-                        emit=emit_new)
-
+new_empty_list_op = c_custom_op(
```

Comment:
Here 'empty' is a bit confusing. What about calling this `new_list_op`, since that's what the underlying function is called?